homework_1/task_02/taxes/testCases_test.go

```diff
@@ -0,0 +1,94 @@
+package taxes_test
+
+import (
+	"code-cadets-2021/homework_1/task_02/taxes"
+	"math"
+)
+
+type testCase struct {
+	inputAmount float64
+	class       []taxes.TaxClass
+	outputTax   float64
+	error       bool
+}
+
+func getTestCases() []testCase {
+	return []testCase{
+		{
+			inputAmount: 7000,
+			class: []taxes.TaxClass{
+				{1000, 0},
```

Comment:
Prefer explicit over implicit attribute initialisation:
```
{ 
UpperThreshold: 1000,
Percentage: 0,
}
```

For example - what happens if you choose to reorder attributes in the `TaxClass` struct later on?